### DISCUSSION TITLE
Timer support in FW

### DISCUSSION THREAD

**Discussion by Paco93** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#28] [Ideas]
Current ESPHOME voice_assistant component supports setting of timers. It would be useful to have such timer supports also in your firmware.
Anyway thanks for your good work on  ESP32_S3Box3 fw.

**Comment by BigBobbas** [comment]
Hi , we are working on it right now :) will announce when we have it sorted

**Comment by Paco93** [comment]
Thanks
